app/assets/images/arrow-down-fill.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M12 16L6 10H18L12 16Z" fill="#08090A"/>
```

Comment:
We can drop the `fill` attribute - it's being overwritten anyway by `crayons-icon` class.